test/common/http/http2/http2_frame.h

```diff
@@ -159,6 +171,13 @@ class Http2Frame {
   void appendData(std::vector<uint8_t> data) {
     data_.insert(data_.end(), data.begin(), data.end());
   }
+  void appendDataAfterHeaders(std::vector<uint8_t> data) {
+    int index = 9;
+    for (uint8_t value : data) {
```

Comment:
could this be consolidated into an `insert` call starting at the 9th index of `data_`? (And is this used?)

Reply:
Used in Metadata frame building. Previously the frame class would allocate null bytes where payload would be, so .appendData would leave a bunch of null bytes in data_.

Reply:
Insert wouldn't work, due to the clause above ^.

Reply:
Will change this to a single line.